sklearn/preprocessing/label.py

```diff
@@ -304,18 +304,41 @@ def fit(self, y):
         self.classes_ = unique_labels(y)
         return self
 
+    def fit_transform(self, y):
+        """Fit label binarizer and transform multi-class labels to binary
+        labels.
+
+        The output of transform is sometimes referred to by some authors as
```

Comment:
I think you can remove the "by some authors". That seems implicit in "sometimes"...